src/app/components/ImageWithCaption/index.tsx

```diff
@@ -0,0 +1,118 @@
+import { createSrcsets } from '../../lib/utilities/srcSet';
+import filterForBlockType from '../../lib/utilities/blockHandlers';
+import { Services } from '../../models/types/global';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import Copyright from '../Copyright';
+import Caption from '../../legacy/containers/Caption';
+import IMAGE from '../Image';
+import styles from './index.styles';
+import { RequestContext } from '../../contexts/RequestContext';
+
+const DEFAULT_IMAGE_RES = 640;
+const LAZYLOAD_FROM_BLOCK = 4;
+
+// @ts-expect-error - TODO: fix types for blocks
+const getText = ({ model }) => model.blocks[0].model.blocks[0].model.text;
+
+const getCopyright = (copyrightHolder: string) => {
+  if (copyrightHolder === 'BBC') {
+    return undefined;
+  }
+
+  return copyrightHolder;
+};
+
+const shouldLazyLoad = (position: number[]) =>
+  !!urlWithPageAnchor() || position[0] > LAZYLOAD_FROM_BLOCK;
+
+const renderCopyright = (copyright: string) =>
+  copyright && <Copyright>{copyright}</Copyright>;
+
+const renderCaption = (block: object, type: string, service: Services) =>
+  // @ts-expect-error - TODO: fix types for blocks
+  block && <Caption block={block} type={type} service={service} />;
+
+type Props = {
+  blocks: object[];
+  position?: number[];
+  sizes?: string;
+  shouldPreload?: boolean;
+};
+
+const ArticleFigure = ({
+  blocks,
+  position = [1],
+  sizes,
+  shouldPreload,
+}: Props) => {
+  const { service } = useContext(ServiceContext);
+  const { isAmp } = useContext(RequestContext);
+
+  if (!blocks) {
+    return null;
+  }
+
+  const rawImageBlock = filterForBlockType(blocks, 'rawImage');
+  const altTextBlock = filterForBlockType(blocks, 'altText');
+  const captionBlock = filterForBlockType(blocks, 'caption');
+
+  const ShouldPreLoadLeadImage =
+    position[0] <= LAZYLOAD_FROM_BLOCK && shouldPreload;
+
+  if (!rawImageBlock || !altTextBlock) {
+    return null;
+  }
+
+  const { locator, originCode, copyrightHolder, height, width } =
+    rawImageBlock.model;
+
+  const alt = getText(altTextBlock);
+
+  const copyright = getCopyright(copyrightHolder);
+
+  const src = buildIChefURL({
+    originCode,
+    locator,
+    resolution: DEFAULT_IMAGE_RES,
+  });
+
+  const { primarySrcset, primaryMimeType, fallbackSrcset, fallbackMimeType } =
+    createSrcsets({
+      originCode,
+      locator,
+      originalImageWidth: width,
+    });
+
+  const lazyLoad = shouldLazyLoad(position);
+
+  return (
+    <figure css={styles.figure}>
+      <IMAGE
+        alt={alt}
+        attribution={copyright}
+        src={src}
+        height={height}
+        width={width}
+        lazyLoad={lazyLoad}
+        preload={ShouldPreLoadLeadImage}
+        srcSet={primarySrcset || undefined}
+        fallbackSrcSet={fallbackSrcset || undefined}
+        mediaType={primaryMimeType || undefined}
+        fallbackMediaType={fallbackMimeType || undefined}
+        sizes={!isAmp ? sizes : undefined}
```

Comment:
Not setting `sizes` on AMP keeps the parity with the old version using the `ImageWithPlaceholder` component.